docs/content_management/work_with_content_types.md

```diff
@@ -35,7 +35,7 @@ Every Content Type has the following general settings:
 |Make content available even with missing translations|When checked, content of this Type will by default be always available, even if it does not have a language version corresponding to the current SiteAccess.|
 
 To add a Field to the new Content Type, select a [Field Type](content_model.md#fields-and-field-types)
-in the Field Type selection drop-down menu and click **Add field definition**.
+in the Field Type selection drop-down menu and click specific **field type**.
```

Comment:
```suggestion
in the Field Type selection drop-down menu and click specific Field Type.
```
We reserve bold for literally quoted part of the interface. Field Type is a domain term which we always capitalize.